pro-duck-store/src/App.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './App.css';
+
+
+import MiniDrawer from "./components/Drawer/drawer";
+import {Store} from "./pages/store";
+import {BrowserRouter, Route, Switch} from "react-router-dom";
+import {Cart} from "./pages/cart";
+import {User} from "./pages/user";
+
+
+function App() {
+  return (
+      <BrowserRouter>
```

Comment:
Here one of the wrappers should be the `<ThemeProvider theme={customTheme}>` which makes your custom theme available for the entire content. Also you should move the `<CssBaseline>` here and also use it as a wrapper.